config/application.rb

```diff
@@ -66,5 +66,10 @@ class Application < Rails::Application
     config.middleware.use 'StatsdMiddleware'
     config.middleware.use 'Rack::Attack'
     config.middleware.use ActionDispatch::Cookies
+    config.middleware.insert_after ActionDispatch::Cookies,
+                                   ActionDispatch::Session::CookieStore,
+                                   key: 'api_session',
+                                   secure: Rails.env.production?,
```

Comment:
might want to make this an environment flag in Settings.yml (I don't really care either way).